run-tests.sh

```diff
@@ -92,6 +93,26 @@ for arch in $ARCHS; do
                 ;;
             esac
         done
+        for test in $TESTS_WINRT; do
+            set +e
+            # compilation should fail for WinRT
+            $arch-w64-$target_os-clang $test.c -o $TEST_DIR/$test.exe -Wimplicit-function-declaration -Werror
+            WINRT_ERROR=$?
+            set -e
+            case $target_os in
+            mingw32winrt)
+                if [ $WINRT_ERROR -eq 0 ]; then
+                    echo "WinRT compilation should have failed for test $test!"
+                    exit 1
+                fi
+                ;;
+            *)
+                if [ $WINRT_ERROR -eq 0 ]; then
+                    TESTS_EXTRA="$TESTS_EXTRA $test"
+                fi
```

Comment:
Same here, please add an else clause to error out if this test doesn't build for the cases where it actually should work.

Reply:
Done